### DISCUSSION TITLE
Data deleted by virus

### DISCUSSION THREAD

**Discussion by Qesrast** [community/community#30764] [Other Feature Feedback, Questions, & Ideas]
hello,please can anyone tell me if there's any way I can get back my data that were deleted by a virus on android phone.
I lost 13 gb's of photos and videos and documents..Please help me I really need those documents back.

**Comment by queenofcorgis** [comment]
Hi @HamzaSR4,

Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on [Android](https://support.google.com/android/community?hl=en). It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.
